sandbox/component/mini-contact/style.scss

```diff
@@ -0,0 +1,47 @@
+@import 'cwel/style/main';
+
+.container--dark {
+    background: #372062;
+}
+
+.mini-contact__button {
+    width: 100%;
+    min-height: 5.5rem;
+    margin-bottom: $s_md;
+    color: $white;
+    text-transform: uppercase;
+    border: 0;
+    border-radius: 4rem;
+
+    &--small { // Modifiers for testing - actual sizes to be created
```

Comment:
@ambtlv What does this mean? When do the actual sizes get created?

Reply:
@Decad - this is just testing and part of the actual component coding. The real buttons created by Nathan will essentially go here. I was just trying to match the UX design...just fillers at this stage.